sentry_sdk/api.py

```diff
@@ -210,5 +211,5 @@ def start_transaction(
     transaction=None,  # type: Optional[Transaction]
     **kwargs  # type: Any
 ):
-    # type: (...) -> Transaction
+    # type: (...) -> Union[Transaction, NoOpSpan]
```

Comment:
does this count as a breaking change then?
In ruby this could always return nil so it wasn't a problem.

Reply:
I think it is not a breaking change. A NoOpSpan has the same interface as a normal Span, but just does nothing. But I think I have to think more about the implications.

Reply:
ok yea I think it should be fine, are there tests for NoOpSpan later in some other PR?

Reply:
Not yet, but I will add some.